src/error.rs

```diff
@@ -38,7 +38,7 @@ pub enum ServerError {
     /// than 0x81 in the response packet.
     BadMagic(u8),
     /// The client did not expect this response from the server.
-    BadResponse(String),
+    BadResponse(Cow<'static, str>),
```

Comment:
I don't know if `Cow<'static, str>` lives with whole process's lifetime? If so, I saw the actual string is came from server, if the server is not a memcached but some server which is using memcached protocol (this use case is real and they had opened some issues before for help), and the error string is a variable, could this will cause some kind of memory leak?

Reply:
No, `Cow<'static, str>` means you can either store a string literal (`&'static str`) or type that can be converted to `str`.